### DISCUSSION TITLE
concordance factor: -scfl flag does not work but -scf does

### DISCUSSION THREAD

**Discussion by crcardenas** [iqtree/iqtree2#366] [Q&A]
Hello, 

I have had this issues running iqtree2 for concordance factors on my datasets. I would like to run the `-scfl` flag, but only the `-scf` flag works with my data. 

I am running `Version IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024`

My command is a simple for loop:
```
for MINTAXA in 27 33 36; do 
iqtree2 -te ethiopia_${MINTAXA}_parti.nex.best_model.nex.treefile \
-s ethiopia_${MINTAXA}min.nex \
-m ethiopia_${MINTAXA}min_parti.nex.best_model.nex \
-T 2 \
--scfl 100 ; 
done
```

When running this code, I get the following error: 
```
...
Reading model parameters from file ethiopia_36min_parti.nex.best_model.nex
ERROR: Expecting a double or a distribution name, but found #nexus. Could not found the distribution named #nexus
```

I have tried loosing the for loop and changing the `-m` flag to `-S` (as seen in another issue: https://github.com/iqtree/iqtree2/issues/155 ; but it is listed as -m on the documentation). However chaning the `-m` flag to `-S`, I get a different error:
```
...
Reading input tree file ethiopia_36min_parti.nex.best_model.nex.treefile ...ERROR: File not found or incorrect input, pls check it again. (line 2 column 0)
```
I even tried the different model files generated by IQTree2 with variations of the `-m` and `-S` flags. So I do not believe that `-m` to `-S` is the solution. 

However, I know these files I am using should work because I have run them using `--scf` which warns me I should use `--scfl` flag when completing the analyses. In both cases using `-m` and `-S` flags work fine for the `--scf` flag. Lastly, I further tried changing my input tree flag from `-te` to `-t` which produces the same errors I've described previously when testing `-t` with the `-m` and `-S` flags.

Here are the associated input files for my smallest dataset [example_scf_error.zip](https://github.com/user-attachments/files/18024270/example_scf_error.zip). Within it are example_scf successful runs and the example `--scfl` error log. Please ignore the quality of the input data, it is fairly "raw" and unprocessed for some issues I have resolved in a parallel dataset.

**Comment by bqminh** [comment]
Sorry for the delay. This is because ethiopia_36min_parti.nex.best_model.nex is a partition file and should be used as such input, see for example http://www.iqtree.org/doc/Advanced-Tutorial#partitioned-analysis-for-multi-gene-alignments. -m is for specifying single models, not partition models.
